json-rpc/types/Quantity.md

```diff
@@ -0,0 +1,33 @@
+---
+title: JSPON RPC Quantity Type Spec
+author: Alita Moore (@alita-moore)
+discussions-to: https://github.com/ethereum-oasis/eth1.x-JSON-RPC-API-standard
+created: 2021-04-13
+---
+
+# `Quantity`
+
+##### Description
+
+`Quantity` represents the 
+
+##### Requirements
+
+| Req. | Description  |
+| ----------- | ------------ |
+|    **Γ1**   | A `Quantity` value **MUST** be hex-encoded |
+|    **Γ2**   | A `Quantity` value **MUST** be "0x"-prefixed |
+|    **Γ3**   | A `Quantity` value **MUST** be expressed using the fewest possible hex digits per byte |
+|    **Γ4**   | A `Quantity` value **MUST** express zero as "0x0" |
```

Comment:
I think I would just prefer a bulleted list here rather than a table.
```suggestion
* A `Quantity` value **MUST** be hex-encoded
* A `Quantity` value **MUST** be "0x"-prefixed
* A `Quantity` value **MUST** be expressed using the fewest possible hex digits per byte
* A `Quantity` value **MUST** express zero as "0x0"
```

Reply:
May want to add a note/line indicating that quantities must be unsigned integers (negative values not supported).

Reply:
May want to mention that leading `0`s should be stripped.